myosuite/envs/myo/myochallenge/run_track_v0.py

```diff
@@ -51,6 +51,12 @@ class RunTrack(WalkEnvV0):
     OSL_PARAM_LIST = []
     OSL_PARAM_SELECT = 0
 
+    # Joint dict
+    pain_jnt = ['hip_adduction_l', 'hip_adduction_r', 'hip_flexion_l', 'hip_flexion_r', 'hip_rotation_l', 'hip_rotation_r',
+                'knee_angle_l', 'knee_angle_l_beta_rotation1', 'knee_angle_l_beta_translation1', 'knee_angle_l_beta_translation2', 
```

Comment:
is the pain measure also needed for the translation joints?

Reply:
I see what you mean. Maybe translation joints are not suitable if we are using joint overextension. I was trying to model the impact pain, for example, stomping the foot or putting all the weight on one leg, would create a large force on the knee in the vertical axis